release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1199,6 +1199,26 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-installer-bug-fixes"]
 ==== Installer
 
+* Previously, if users specified an {product-title} cluster name with a dot (.), the installation failed. This update adds a validation check to the installation program, which returns an error if a dot is present in the cluster name. link:https://bugzilla.redhat.com/show_bug.cgi?id=1989724[*BZ#2084580*]
```

Comment:
Update the BZ ID in the link

Reply:
Is it worth figuring out what happens if you use a dot in the name? It appears that you can re-renter the name immediately. But not sure. 
So then we could: "As a result, you can re-enter a valid name immediately, before the installation proceeds."